pages/linux/modinfo.md

```diff
@@ -0,0 +1,11 @@
+# modinfo
+
+> Extract information about a Linux kernel module.
+
+- List all attributes of a kernel module:
+
+`modinfo {{kernel_module}}`
+
+- List the specified attribute(s) only:
+
+`modinfo -F {{author|description|license|parm|filename}} {{kernel_module}}`
```

Comment:
It's a bit ambiguous here how multiple attributes are delimited. Is it with the vertical bar?

Reply:
Yes the vertical bar signifies multiple values which can be used. It's what we have been using.

Reply:
Ah no, I don't think so. ping @aydwi

Reply:
Actually, multiple attributes can be specified, separated by a space.

Reply:
Ok, then in that case change it to - `List the specified attribute(s) only. (You can specify multiple attributes separated by space)`

Reply:
After some testing, it looks like it does not accept multiple attributes in a single command. I've fixed the mistake and pushed again.